ambry-replication/src/main/java/com/github/ambry/replication/RemoteReplicaInfo.java

```diff
@@ -62,6 +62,10 @@
   private long reEnableReplicationTime = 0;
   private ReplicaThread replicaThread;
 
+  // Metadata response information (contains missing blobs, remote token, local lag from remote, etc.) of previous replication cycle.
```

Comment:
Minor: the comment looks like super long. There is a line in IDE, usually we don't code or comment after that line.